.secrets.baseline

```diff
@@ -109,6 +109,36 @@
     },
     {
       "path": "detect_secrets.filters.heuristic.is_templated_secret"
+    },
+    {
+      "path": "detect_secrets.filters.regex.should_exclude_file",
+      "pattern": [
+        ".*/TestFixtures.java$"
```

Comment:
Excluding this file since all of these are used for testing but wondering if I should use inline exclusions here instead for added safety?

Reply:
Good question. Excluding by file feels a little risky - I could certainly see a scenario where a dev replaces a test VC with a prod one and uses a unit test as a way of running the decryption.

(but if it's really painful then this might be risk we want to accept)